EU4toV2/Data_Files/configurables/dead_definitions.txt

```diff
@@ -91,6 +92,7 @@ link = { tag = BUG primary_culture = baganda capital = 2019 }
 link = { tag = CAS capital = 78 government = democracy civilized = yes }
 link = { tag = CAY government = democracy civilized = yes capital = 2241 }
 link = { tag = CEG capital = 1751 civilized = yes }
+link = { tag = CEL primary_culture = irish government = absolute_monarchy civilized = yes capital = 300 add_accepted_culture = welsh add_accepted_culture = pictish add_accepted_culture = breton }
```

Comment:
add_accepted_culture is not a valid flag for this file.

Reply:
So I should add through Taiping-n-CSA? It is accepted by Vic2 syntax
![image](https://user-images.githubusercontent.com/24550392/115141849-cf397d80-a03e-11eb-8206-d3c0035e5b36.png)